api/envoy/config/filter/udp/dns_filter/v2alpha/dns_filter.proto

```diff
@@ -5,6 +5,8 @@ package envoy.config.filter.udp.dns_filter.v2alpha;
 import "envoy/api/v2/core/base.proto";
 import "envoy/data/dns/v2alpha/dns_table.proto";
 
+import "google/protobuf/duration.proto";
```

Comment:
Sure, I can work on that.  I've already updated the implementation locally to use the v3alpha generated code.  Let me see what I can get done here.

Reply:
OK thanks I will take a look when this is complete.

/wait

Reply:
This is complete.  Please let me know if you see anything else that should be done.  I did see the v2alpha file is `FROZEN` so any changes going forward will be done in the `v3alpha` proto files.

Reply:
You are still making changes to the v2alpha protos here. TBH, since the filter is not yet functional, I think it would be better to just delete v2alpha entirely and move straight to v3 (we just did this for the ongoing WASM work)? I think that will make this simpler. Can you go ahead and do that?

Reply:
Sorry I didn't unresolve this. Ping on this in case you missed it.

/wait

Reply:
I removed v2alpha to the proto for the DNS Filter exists only in v3alpha.